src/static/static_context.rs

```diff
@@ -30,6 +38,13 @@ impl FutharkContext {
         }
     }
 
+    pub fn free(&mut self) {
```

Comment:
I think we should add a note that this will cause any new calls using this context to fail.

Reply:
Should `free` take `Self` as an argument, so that the context cannot be used after having been freed?

Reply:
Yes, but that is outside the scope of this pr as it would require changes such as the ones suggested in https://github.com/Erk-/genfut/pull/20